Assets/__Scripts/MapEditor/UI/Strobe Generator/StrobeTextField.cs

```diff
@@ -0,0 +1,8 @@
+﻿using TMPro;
+using UnityEngine;
+
+[RequireComponent(typeof(TMP_InputField))]
+public class StrobeTextField : DisableActionsField
```

Comment:
Honestly, this seems like something useful enough to where renaming into something like `DisableActionsOnInteract` would make sense.

Reply:
The point is that the DisableActionsField class calls `DisableActionMaps` based on a call to `GetType()`, so it's a separate class on purpose to isolate these from other things that call that method